profiles/common/modules/features/nexteuropa_editorial/nexteuropa_editorial.install

```diff
@@ -91,7 +91,6 @@ function nexteuropa_editorial_install() {
   multisite_config_service('title')->replaceTitleField('node', 'editorial_team', 'title');
 
   // Add NextEuropa specific moderation states.
-  multisite_config_service('workbench_moderation')->createModerationState('validated', 'Validated', 'Ready to be reviewed', 0);
```

Comment:
If we removed this line, should we not add a dependency to multisite_rules_configurations?